test/functional/mining_basic.py

```diff
@@ -55,8 +55,16 @@ def mine_chain(self):
         assert_equal(mining_info['blocks'], 200)
         assert_equal(mining_info['currentblocktx'], 0)
         assert_equal(mining_info['currentblockweight'], 4000)
+
+        self.log.info('test blockversion')
+        self.restart_node(0, extra_args=['-mocktime={}'.format(t), '-blockversion=1337'])
+        self.connect_nodes(0, 1)
+        assert_equal(1337, self.nodes[0].getblocktemplate(NORMAL_GBT_REQUEST_PARAMS)['version'])
+        self.restart_node(0, extra_args=['-mocktime={}'.format(t)])
+        self.connect_nodes(0, 1)
+        assert_equal(0x30000000, self.nodes[0].getblocktemplate(NORMAL_GBT_REQUEST_PARAMS)['version'])
```

Comment:
nit: To me it wasn't obvious at first why the version now is `0x30000000`. Would it make sense to extract this "magic value" into a variable called `versionTaproot` for example?

Reply:
This has nothing to do with taproot. Changed to `VERSIONBITS_TOP_BITS + (1 << VERSIONBITS_DEPLOYMENT_TESTDUMMY_BIT)`